apps/judicial-system/backend/src/app/modules/case/filters/case.filter.ts

```diff
@@ -226,6 +226,11 @@ const canPrisonAdminUserAccessCase = (
   theCase: Case,
   forUpdate: boolean,
 ): boolean => {
+  //Allow prison admin to update isRegisteredInPrisonSystem
+  if (theCase.type === CaseType.INDICTMENT) {
+    return true
+  }
+
```

Comment:
This gives prison admin access to all indictment cases, which is worng.

This can be fixed by moving the logic into the `forUpdate` condition below, like this:

```
if (forUpdate) {
  if (!isIndictmentCase(theCase.type) {
    return false
  }
}
```

The method ```isIndictmentCase``` needs to be imported from the type library.